efi/preinstall/errors.go

```diff
@@ -20,14 +20,57 @@
 package preinstall
 
 import (
+	"bufio"
 	"bytes"
 	"errors"
 	"fmt"
+	"io"
 
 	"github.com/canonical/go-tpm2"
 	internal_efi "github.com/snapcore/secboot/internal/efi"
 )
 
+// indentLines is a helper for managing indenting in nested multi-line
+// errors.
+func indentLines(n int, str string) string {
+	r := bytes.NewReader([]byte(str))
+	w := new(bytes.Buffer)
+	br := bufio.NewReader(r)
+	for {
+		line, err := br.ReadString('\n')
+		fmt.Fprintf(w, "%*s%s", n, "", line)
```

Comment:
should we really do this print if line is "" and err is io.EOF ?

Reply:
I've refactored this function a bit (it uses `bufio.Scanner` now and it's been renamed to make it clearer what its intended use is.